### PR TITLE
Release 1.14 Preparation

### PR DESCRIPTION
**Pull request by hisarbalik**
**Description**

Changes proposed in this pull request:

- #8981
- #8988 
- #9014 
- #9007 
- #8989 

**Related issue(s)**

### CODE REVIEW

**Comment by ebensom** [comment]
Hi @hisarbalik ,

Please also cherry-pick https://github.com/kyma-project/kyma/pull/9014

Thanks much!

File: tools/kyma-installer/kyma.Dockerfile [hunk 1]
```diff
@@ -1,6 +1,4 @@
-ARG INSTALLER_VERSION="98e02519"
-ARG INSTALLER_DIR=eu.gcr.io/kyma-project
-FROM $INSTALLER_DIR/kyma-operator:$INSTALLER_VERSION
+FROM eu.gcr.io/kyma-project/kyma-operator:master-98e02519
```
Comment (adamwalach) [inline tools/kyma-installer/kyma.Dockerfile:1]: This image is not present in gcr: https://console.cloud.google.com/gcr/images/kyma-project/EU/kyma-operator?gcrImageListsize=30
But `eu.gcr.io/kyma-project/kyma-operator:98e02519` is, so I think we should change that.

**Comment by hisarbalik** [comment]
Run /test pre-re114-kyma-artifacts

**Comment by k15r** [comment]
why don't you cherrypick those PRs directly onto the branch, and instead cherrypick them onto your PR? Directly onto the release branch would make it more obvious which changes are included in the release.

File: docs/serverless/05-01-serverless-envs.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+To configure the Serverless Function, override the default values of these environment variables:
+
+| Environment variable | Description                                                                   | Unit   | Default value |
+| ---------------------- | ----------------------------------------------------------------------------- | ------ | ------------- |
+| **FUNC_TIMEOUT**       | Specifies the number of seconds in which a runtime must execute the code. | Number | `180`           |
+| **REQ_MB_LIMIT**       | Specifies the payload body size limit in megabytes.                                          | Number |  `1`            |
+
+See [`kubeless.js`](https://github.com/kubeless/runtimes/blob/master/stable/nodejs/kubeless.js) to get a deeper understanding of how the Express server, that acts as a runtime, uses these values internally to run Functions.
```
Comment (majakurcius) [inline docs/serverless/05-01-serverless-envs.md:13]:
```suggestion
See [`kubeless.js`](https://github.com/kubeless/runtimes/blob/master/stable/nodejs/kubeless.js) to get a deeper understanding of how the Express server, which acts as a runtime, uses these values internally to run Functions.
```

**Comment by Disper** [comment]
This PR was force-merged. See the reasoning in following thread on open-source slack: https://kyma-community.slack.com/archives/CPVGXNT61/p1595323646046400?thread_ts=1593175150.034400&cid=CPVGXNT61

**Pull request merged**